src/osemgrep/networking/Scan_helper.mli

```diff
@@ -0,0 +1,11 @@
+(* from scans.py *)
+
+val start_scan :
```

Comment:
maybe a small comment explaining what it does.

Reply:
done in b8f3090fd